### PR TITLE
[Fixed Bug] Data Visualization with D3: Learn About SVG in D3

### LINKED ISSUE #20384: [Bug] Data Visualization with D3: Learn About SVG in D3
**Describe your problem and how to reproduce it:**
The following code doesn't pass the test.
```js
const svg = d3.select("body")
                  // Add your code below this line
        .append("svg")
        .style("width", w)
        .style("height", h)   
```
but the following does:
```js
const svg = d3.select("body")
                  // Add your code below this line
        .append("svg")
        .attr("width", w)
        .attr("height", h)    
```
### Why should the first snippet pass the test? 

Because `style()` and `attr()` method works the same way in case of `height` and `width`. The `style()` method has been taught in previous lesson and we shouldn't restrict the solution to just 1 way.

Although in the description of the challenge, it's explicitly mentioned to use `.attr()` method, I suppose we'll need to update this text as well.
>Give it a width attribute set to the provided w constant and a height attribute set to the provided h constant using the attr() method for each.

### Resolve:
This could be resolved by updating the text in the description and the test to something like:
```js
$("svg").attr("width") == "500" || $("svg").style("width") == "500"
```
Likewise for height.

**Add a Link to the page with the problem:**

https://learn.freecodecamp.org/data-visualization/data-visualization-with-d3/learn-about-svg-in-d3

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: 70
* Operating System: Windows 10

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #20384

### CODE REVIEW

**Comment by thecodingaviator** [comment]
@Asjas

**Comment by thecodingaviator** [comment]
Closed/reopened to clear Travis CI error

**Comment by scissorsneedfoodtoo** [comment]
@thecodingaviator, sorry for the delay in reviewing this. There was some problem with D3 being loaded to the page that just got sorted out over the last few days.

This looks good to me and works locally. I just have one suggestion for simplifying the tests a bit.

File: curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/learn-about-svg-in-d3.english.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ Add an <code>svg</code> node to the <code>body</code> using <code>append()</code
 tests:
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/learn-about-svg-in-d3.english.md]: Could we change the tests to the following? Just seems to be a bit more readable:

```
tests:
  - text: Your document should have 1 <code>svg</code> element.
    testString: assert($('svg').length == 1, 'Your document should have 1 <code>svg</code> element.');
  - text: The <code>svg</code> element should have a <code>width</code> attribute set to 500 or styled to have a width of 500px.
    testString: assert($('svg').attr('width') == '500'||$('svg').css('width') == '500px', 'The <code>svg</code> element should have a <code>width</code> attribute set to 500 or styled to have a width of 500px.');
  - text: The <code>svg</code> element should have a <code>height</code> attribute set to 100 or styled to have a height of 100px.
    testString: assert($('svg').attr('height') == '100'||$('svg').css('height') == '100px', 'The <code>svg</code> element should have a <code>height</code> attribute set to 100 or styled to have a height of 100px.');
```

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/learn-about-svg-in-d3.english.md (hunk 1 above)
Comment (thecodingaviator) [reply curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/learn-about-svg-in-d3.english.md]: Alright, done

**Comment by scissorsneedfoodtoo** [comment]
@thecodingaviator, awesome! Everything LGTM. Thanks for your contribution!

**Pull request merged**